pkg/version/version.go

```diff
@@ -0,0 +1,207 @@
+package version
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	goversion "github.com/hashicorp/go-version"
+	pluginutils "github.com/storageos/kubectl-storageos/pkg/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	operatorLatestSupportedVersion = "v2.5.0"
```

Comment:
Am i right we have to build a separated binary for each version while mostly the logic don't change?